### PR TITLE
K8SPG-467 - Release 1.5.0

### PR DESCRIPTION
**Pull request by ptankov**
[K8SPG-467](https://jira.percona.com/browse/K8SPG-467) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are the manifests (crd/bundle) regenerated if needed?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PG version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

File: deploy/cr-minimal.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
   port: "5432"
   user: pguser
   pgPrimary:
-    image: perconalab/percona-postgresql-operator:main-ppg14-postgres-ha
+    image: perconalab/percona-postgresql-operator:1.5.0-ppg14-postgres-ha
```
Comment (tplavcic) [inline deploy/cr-minimal.yaml:12]: `perconalab`->`percona`, same for others

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Review by hors — DISMISSED** [review]

**Review by inelpandzic — DISMISSED** [review]
@ptankov pls just update PR description.

File: e2e-tests/functions [hunk 2]
```diff
@@ -389,7 +393,7 @@ wait_job_completion() {
 		fi
 	done
 	echo
-	set -o xtrace
+	# set -o xtrace
```
Comment (hors) [inline e2e-tests/functions:396]: why do we need to have it?

Comment (ptankov) [reply e2e-tests/functions:396]: We don't. It was temporary for debugging purposes. Thanks for catching this!

File: e2e-tests/functions [hunk 3]
```diff
@@ -971,14 +973,26 @@ run_psql_ssl() {
...
+	local client_container=$(kubectl_bin get pods --selector=name=pg-client -o 'jsonpath={.items[].metadata.name}')
 	local sslmode="$5"
 
-	kubectl_bin exec ${client_container} -n $namespace -- \
-		bash -c "printf '$command\n' | PGSSLMODE=$sslmode PGSSLROOTCERT=/tmp/tls/ca.crt psql -v ON_ERROR_STOP=1 -t -q $driver://$uri$suffix" || true
+	kubectl_bin exec ${client_container} -- \
+		bash -c "printf '$command\n' | PGSSLMODE=$sslmode PGSSLROOTCERT=/tmp/tls/ca.crt psql -v ON_ERROR_STOP=1 -t -q $driver://$uri" || true
 }
 
+# run_psql_ssl() {
```
Comment (hors) [inline e2e-tests/functions:984]: can we remove it or uncomment?

Comment (ptankov) [reply e2e-tests/functions:984]: sure!

**Review by hors — APPROVED** [review]

**Pull request merged**